### PR TITLE
Achtung Cthulhu 2d20

### PR DESCRIPTION
**Pull request by lozanoje**
Support for Achtung Cthulhu 2d20

https://github.com/Muttley/foundryvtt-ac2d20

### CODE REVIEW

File: scripts/damage-log.js [hunk 1]
```diff
@@ -144,6 +144,25 @@ class DamageLog {
...
+				value: "injuries.value"
+			},
+			fortune: {
+				invert: false,
+				value: "fortune.value"
+			},
+			fatigue: {
+				invert: true,
+				value: "fatigue"
+			}
```
Comment (cs96and) [inline scripts/damage-log.js:165]: Can you move this so the list is in alphabetical order please?

File: module.json [hunk 2]
```diff
@@ -99,6 +100,10 @@
 			{
 				"id": "worldbuilding",
 				"type": "system"
+			},
+			{
+				"id": "ac2d20",
+				"type": "system"
```
Comment (cs96and) [inline module.json:106]: Can you move this so the list in is alphabetical order please?

**Review by cs96and — CHANGES_REQUESTED** [review]
Thanks for this.  See my comments below.  Can you also update the list of supported systems in the README.md please?

File: scripts/damage-log.js (hunk 1 above)
Comment (lozanoje) [reply scripts/damage-log.js]: Fixed: https://github.com/cs96and/FoundryVTT-damage-log/pull/42/commits/3922d88427048bcf4e1794a1891c895a724f4820

File: module.json (hunk 2 above)
Comment (lozanoje) [reply module.json]: Fixed: https://github.com/cs96and/FoundryVTT-damage-log/pull/42/commits/3922d88427048bcf4e1794a1891c895a724f4820

File: module.json [hunk 3]
```diff
@@ -29,17 +29,20 @@
...
 		"pf1",
 		"pf2e",
-		"worldbuilding",
-		"archmage",
 		"shaper",
-		"ac2d20"
+		"sw5e",
+		"swade",
+		"tormenta20",
+		"worldbuilding"
```
Comment (cs96and) [inline module.json:45]: Thanks, didn't realize this was already out of order!

**Review by cs96and — APPROVED** [review]
Looks good, thanks!

**Comment by cs96and** [comment]
This is now in v1.18.0

**Pull request merged**